assets/scss/style.scss

```diff
@@ -27,25 +27,17 @@ $font-family-monospace:  {{ .Site.Params.mono_font | default "SFMono-Regular, Me
 // Structure
 @import "custom/structure/general";
 @import "custom/structure/topbar";
-// @import "custom/structure/content";
-// @import "custom/structure/sidebar-layouts";
-// @import "custom/structure/doc-nav";
-// @import "custom/structure/toc-layouts";
 @import "custom/structure/footer";
 
-// // Components
+// Components
 @import "custom/components/buttons";
 @import "custom/components/badge";
-@import "custom/components/backgrounds";
```

Comment:
IMHO, this should not be removed. This breaks the background-color of the nav-pills of the image-comparer and maybe other things as well